src/index.js

```diff
@@ -268,6 +268,10 @@ module.exports = {
     expect = expect.child();
     expect.use(require('magicpen-prism'));
 
+    // export the items registered by magicpen-prism to callers
+    expect.exportTheme('html', expect.output.theme('html'));
+    expect.exportStyle('code', expect.output.styles.code);
```

Comment:
Hmm, this is a bit awkward. I wonder if we should make theme-exporting a magicpen feature?

Reply:
@papandreou hm I’m not sure it makes sense for magicpen - it doesn’t have a concept of children, so we’d end up with it having an exportTheme method being synonymous with installTheme which Unexpected core would overwrite to treat specially.

I tried to note down the constraints that played in here in the description, but to summarise exporting is an Unexpected only concept and my sense is it should remain so. I’d also note exportStyle already works this way, so it seemed consistent.

Reply:
But it's nasty if every plugin that uses magicpen-prism internally has to do this.

Reply:
@papandreou I see what you're getting at - may have a suggestion.